src/azure-cli/azure/cli/command_modules/iot/_params.py

```diff
@@ -96,8 +96,16 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
                        arg_type=get_enum_type(AccessRightsDescription),
                        help='Access rights for the IoT Hub Device Provisioning Service. '
                             'Use space-separated list for multiple rights.')
-            c.argument('primary_key', help='Primary SAS key value.')
-            c.argument('secondary_key', help='Secondary SAS key value.')
+
+    with self.argument_context('iot dps policy create') as c:
+        c.argument('primary_key', help='Primary SAS key value.')
```

Comment:
Should we append, if not provided will be generated by the service?

Reply:
That's a good idea, this way we're consistent :)